### ISSUE TITLE
Fields Method Add-On

### ISSUE THREAD

**Issue by tuxgoose** [AtlassianPS/JiraPS#292]
Ability to limit data fields from a jiraIssue
### Expected Behavior

Get-JiraIssue -Fields "assignee,labels,description" -Key example-45

### Current Behavior

Get-JiraIssue  -Key example-45 
-currently you return every field within the jira issue

Get-JiraIssue -Fields "assignee,labels,description" -Key example-45
-would only return the fields given
ID          : 57538
labels      : testing
Assignee    : Unassigned
RestUrl     : https://website.com/rest/api/latest/issue/example-45
Key         : example-45
HttpUrl     : https://website.com/browse/example-45
Description :example description of story
Status      :
Summary     :
Transition  : {}

### Possible Solution

check for the fields parameter and modify the resource uri if present

### Context

We have some hundred of additional customfields in jira that makes requesting issues very slow

### Your Environment

> ```powershell
> Get-Module JiraPS -ListAvailable | select version
> $PSVersionTable
> ```
JiraPS version 2.7.0
PSVersion  5.1.16299.431

**Comment by tuxgoose** [comment]
@lipkau  could you assign this to me?

**Comment by tuxgoose** [comment]
@lipkau when the code get merged with master do we delete this issue?

**Comment by lipkau** [comment]
correct :-)

**Comment by tuxgoose** [comment]
Was there an issue with the code?

**Comment by lipkau** [comment]
no. I am waiting for microsoft to fix a problem we have with azure devops.
Then I can migrate the CI and release this feature

**Comment by lipkau** [comment]
related to #62

**Comment by lipkau** [comment]
Done — `Get-JiraIssue -Fields [String[]]` exists and defaults to `*all`. Example: `Get-JiraIssue TEST-1 -Fields assignee,labels,description`. Closing.
